cloud/pkg/dynamiccontroller/application/application.go

```diff
@@ -441,22 +455,71 @@ func (c *Center) Process(msg model.Message) {
 	klog.Infof("[metaserver/applicationCenter]successfully to process Application(%+v)", app)
 }
 
+func (c *Center) createAuthClient(app *Application) (authorizationv1client.AuthorizationV1Interface, error) {
+	parts := strings.SplitN(app.Token, " ", 3)
+	if len(parts) < 2 || strings.ToLower(parts[0]) != "bearer" {
+		return nil, fmt.Errorf("invalid request token format or length: %v", len(parts))
+	}
+
+	token := parts[1]
+	c.authConfig.BearerToken = token
```

Comment:
When edge nodes issue a large number of list-watch requests at the same time ,  for example, when the cloudcore is upgrade or the edge node reconnect to the cloud at the same time, It is possible to write BearerToken at the same time and the authConfig.BearerToken maybe dirty and the result is unbelievable，There is even a problem of escalation of rights

Reply:
`createAuthClient` will new a temporary `kubeconfig` every time when accepting an application message. It is concurrent.

Reply:
Done. A DeepCopy operation can prevent the escalation of rights.